posts/deep_work.md

```diff
@@ -0,0 +1,35 @@
+1. An increase in the quality of your output;
+2. An increase in the quantity of your output; and 
+3. A passion that manifests as a deeper sense of satisfaction when undertaking and completing tasks.
+
+In a 24/7 world driven by technological change and innovation, distractions are everywhere and unavoidable. Emails, social media interaction and digitalisation of basic administrative tasks often require immediate attention and an instantaneous response. The workspace, whether office-based or remote, as a result, becomes a hot-bed of multi-tasking, interruption, and profound information overload. And when it’s been [found](https://www.ics.uci.edu/~gmark/chi08-mark.pdf) that re-focusing on a task after being distracted takes a person on average, 23 minutes and 15 seconds, this just leads to more speed and more stress. 
+
+Like a lock that eventually gives way after being repeatedly picked at, unconstructive, fragmented time reduces the ability to concentrate and therefore resist such distractions. If permitted to continue then over time, deeper working becomes harder to attain and you need to work for an increasing amount of time on complex tasks before reaching peak productivity.
+
+Consequently, at Made Tech, we’ve been trying a number of techniques to help teams commit to periods of deep working. Success of each varies from person-to-person as everyone responds differently to external stimuli and some are naturally better than others at zoning out distractions. While it’s worth taking time to work out what works on an individual level, these are some strategies we would recommend. 
+
+## Adapt your working environment
+
+Flexible working is embedded in our culture so people can choose where they work from and largely when. We do however try to discourage working in the evenings to ensure work/life balance, and because it has the potential to prevent relaxation and risk negatively impacting work being done the following day.     
+
+## Time yourself
+
+During pairing activities our teams use [pomodoros](http://tomatotimers.com/) - a time management method which breaks down work into intervals of 25 minutes in length, separated by short five minute breaks.  Our developers and engineers have also used the timer system while working on client projects with good outcomes, so recently we extended its usage to the marketing and sales teams as well. 
+
+## Make it a daily habit
+
+Planning periods of deep work and creating a ritual makes it easier to focus than when you try to transition from shallow work when the moment (or a deadline) takes you. Being in a quiet space at a set time every day gets you into a rhythm and stops energy from being expended unnecessarily by jumping between complex and simpler tasks.  
+
+From experience, we’ve found that deep work is best practiced in the morning when distractions are less frequent, or the early afternoon prior to post-lunch slumps.  
```

Comment:
Is there a source for this?

Reply:
no, it simply lies in the fact that you re less likely to be interrupted early in the work day, as everyone is getting started